### PR TITLE
Add tests for warnings

### PR DESCRIPTION
**Pull request by cameronbowie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ $ ./launch.sh
 
 Once the server is running, you can run the mock API tests for your SDK. See the SDK's documentation for more information.
 
+You can check which scenarios are included in the package by referencing the README in the package. This README includes information on how to run the server as well as descriptions of each scenario.
```
Comment (stollgr) [inline README.md:32]: Great addition of information for the user.

File: lib/clean_postman_scenario.js [hunk 2]
```diff
@@ -101,11 +102,11 @@ function warnAboutResponse(scenarioName, response) {
...
-    var postmanParamVars = str.match(/:[^{}\/]+\//g);
+    var postmanParamVars = str.match(/:[^{}\/]+/g);
     _.each(postmanVars, (v) => uncleanWarning(scenarioName, "uses Postman variable, '" + v + "'"));
-    _.each(postmanParamVars, (v) => uncleanWarning(scenarioName, "uses Postman variable, '" + v.slice(0, -1) + "'"));
+    _.each(postmanParamVars, (v) => uncleanWarning(scenarioName, "uses Postman variable, '" + v + "'"));
 }
 
 function uncleanWarning(scenarioName, message) {
-    console.warn("WARNING: Scenario '" + scenarioName + "': " + message)
+    console.error("WARNING: Scenario '" + scenarioName + "': " + message)
```
Comment (stollgr) [inline lib/clean_postman_scenario.js:123]: Is this so the unit tests will capture the warning?

Comment (cameronbowie) [reply lib/clean_postman_scenario.js:123]: Sort of. When I was looking into how to capture warnings, I found that `console.warn()` is just an alias for `console.error()` so I changed it to be more clear what was going on

Comment (stollgr) [reply lib/clean_postman_scenario.js:123]: Great. Thank you for explaining.

**Review by stollgr — APPROVED** [review]

**Pull request merged**